modules/tensor_mechanics/doc/content/documentation/systems/Materials/ComputeStrainIncrementBasedStress.md

```diff
@@ -1,14 +1,33 @@
-<!-- MOOSE Documentation Stub: Remove this when content is added. -->
+# Compute Strain Increment Based Stress
 
-# ComputeStrainIncrementBasedStress
+!syntax description /Materials/ComputeStrainIncrementBasedStress
 
-!alert construction title=Undocumented Class
-The ComputeStrainIncrementBasedStress has not been documented, if you would like to contribute to MOOSE by
-writing documentation, please see [/generate.md]. The content contained on this page explains
-the typical documentation associated with a MooseObject; however, what is contained is ultimately
-determined by what is necessary to make the documentation clear for users.
+## Description
 
-!syntax description /Materials/ComputeStrainIncrementBasedStress
+This stress calculator finds the value of the stress as a function of the elastic
+strain increment when a series of inelastic strains are specified in the input file.
+The stress is calculated as
+\begin{equation}
+  \label{eqn:stress}
+  \sigma_{ij} = \sigma_{ij}^{old} + C_{ijkl} \Delta \epsilon_{jk}^{el}
```

Comment:
It doesn't seem to currently have any effect, but I think we should be using \boldsymbol on all these tensors.

Reply:
These ones have indices though...I'd propose that only tensor quantities without indices should have bold tensor added. Sound okay?

Reply:
Oh yeah, you're right.